packages/cli/ts/index.ts

```diff
@@ -560,10 +560,10 @@ program
         pollStateTreeDepth: args.pollStateTreeDepth || args.stateTreeDepth,
         pollJoiningVerifyingKey: pollJoiningVerifyingKey!,
         pollJoinedVerifyingKey: pollJoinedVerifyingKey!,
-        processMessagesVerifyingKey: processVerifyingKey!,
-        tallyVotesVerifyingKey: tallyVerifyingKey!,
+        processMessagesVerifyingKeys: [processVerifyingKey!],
+        tallyVotesVerifyingKeys: [tallyVerifyingKey!],
         verifyingKeysRegistryAddress,
-        mode: args.useQuadraticVoting ? EMode.QV : EMode.NON_QV,
+        modes: args.useQuadraticVoting ? [EMode.QV] : [EMode.NON_QV],
```

Comment:
I think here it can support multiple keys

Reply:
I created a new issue related to it 😊 : https://github.com/privacy-scaling-explorations/maci/issues/2410